app/components/UserContributions/ContributionCard.js

```diff
@@ -275,10 +275,9 @@ export default class ContributionCard extends React.Component {
               : null}
             {!isPending ? (
               <TextSpan>
-                {' '}
                 {cardType && cardType.length > 0
-                  ? cardType.charAt(0).toUpperCase() + cardType.slice(1)
-                  : ''}
+                  ? ' ' + cardType.charAt(0).toUpperCase() + cardType.slice(1)
```

Comment:
I am not sure which is best let me know 
can we use here template string
so we no need to concate using + sign

```suggestion
                  ? ` ${cardType.charAt(0).toUpperCase()} ${cardType.slice(1)}`
```

Reply:
I have no clue what is the better solution, but there shouldn't be spaces between the capital letter the rest of the word or you would see "D onation" and "G ift" in the list :-)

Reply:
Maybe the `+` is only doing string concatenation while the other is using a template engine parser? Then I would prefer simple string concatenation, but I am not sure.